### ISSUE TITLE
can't test send mail using snmp

### ISSUE THREAD

**Issue by lamhaison** [giuseppeiannelli/docker-cacti#1]
When I send mail with snmp protocol with my configuration:
``
This is the hostname/IP of the SMTP Server you will send the email to.
```
SMTP Hostname
This is the hostname/IP of the SMTP Server you will send the email to.
ssl://smtp.gmail.com
SMTP Port
This is the port on the SMTP Server that SMTP uses.	
465
```
And error when test send email
```
ERROR: Unable to connect to SMTP Host 'ssl://smtp.gmail.com': (32730) Unable to find the socket transport "ssl" - did you forget to enable it when you configured PHP?
```

I make sure that username and password are correct. Do you test with sending email with snmp protocol.

How to fix it?, I have spent for 4 hours to fix it but I didn't find any way. Thanks

**Comment by giuseppeiannelli** [comment]
SNMP is not a protocol to send email.
If you want send email to notify host down or error, you must set your SMTP server.

**Comment by giuseppeiannelli** [comment]
Sorry, I've forgotten to install openssl package and php-openssl module.
Now I added it and build the new docker image on docker hub https://hub.docker.com/r/giuseppeiannelli/cacti/

**Comment by lamhaison** [comment]
I have misstake smtp not snmp
Vào 12-02-2017 04:00, "Giuseppe Iannelli" <notifications@github.com> đã
viết:

**Comment by lamhaison** [comment]
After install some ssl packages. I can configure ssl but when run test send mail i see error
ERROR: Unable to connect to SMTP Host 'ssl://smtp.gmail.com': (0)

I can't tenlet to port with commandline (telnet smtp.gmail.com 465). I make sure that container can access internet such as install package with internet, wget file. I don't understand what's master with your container.
Dou you have tested send mail with you container ever?

**Comment by giuseppeiannelli** [comment]
Yes, I try to send email with mandrill smtp server and I have success.
![screen shot 2017-02-12 at 14 55 38](https://cloud.githubusercontent.com/assets/7208198/22862681/af9b8ea0-f133-11e6-9d5c-7ea1c73e1523.jpg)

**Comment by lamhaison** [comment]
Do you have tested with gmail server ever?. I can't configure with gmail server :(. But with other cacti docker of other somebody then It's ok.

**Comment by giuseppeiannelli** [comment]
I have never configure smtp with google with success.
When I have tried, the mail test return an error with no description or code.

**Comment by lamhaison** [comment]
Do you have any solution :(?

**Comment by giuseppeiannelli** [comment]
Sorry, I have not any solution with Google SMTP
